modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,239 @@
+  namespace: ibi-ns
+spec:
+  bareMetalHostRef:
+    name: ibi-bmh <2>
+    namespace: openshift-machine-api
+  clusterDeploymentRef:
+    name: ibi-cluster-deployment <3>
+  hostname: ibi-host <4>
+  imageSetRef:
+    name: ibi-image-install <5>
+  machineNetwork: 10.0.0.0/24 <6>
+----
+<1> Specify the name for the `ImageClusterInstall` resource.
+<2> Specify the `BareMetalHost` resource that you want to target for the image-based installation.
+<3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
+<4> Specify the host name for the cluster.
+<5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
+<6> Specify the public CIDR (Classless Inter-Domain Routing) of the external network.
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f ibi-image-cluster-install.yaml.yaml
+----
+
+. Create the `ClusterDeployment` resource:
+
+.. Create a YAML file that defines the `ClusterDeployment` resource:
++
+.Example `ibi-cluster-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: ibi-cluster-deployment <1>
+  namespace: ibi-ns <2>
+spec:
+  baseDomain: example.com <3>
+  clusterInstallRef:
+    group: extensions.hive.openshift.io
+    kind: ImageClusterInstall 
+    name: ibi-image-install <4>
+    version: v1alpha1
+  clusterName: ibi-cluster <5>
+  platform:
+    none: {}
+  pullSecretRef:
+    name: pull-secret <6>
+----
+<1> Specify the name for the `ClusterDeployment` resource.
+<2> Specify the namespace for the `ClusterDeployment` resource.
+<3> Specify the base domain that the cluster should belong to.
+<4> Specify the name of the `ImageClusterInstall` in which you defined the container images to use for the image-based installation of the target host.
+<5> Specify a name for the cluster.
+<6> Specify the deployment platform for the cluster.
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-deployment.yaml
+----
+
+. Create the `ManagedCluster` resource:
+
+.. Create a YAML file that defines the `ManagedCluster` resource:
++
+.Example `ibi-managed.yaml` file
+[source,yaml]
+----
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: sno-ibi <1>
+spec:
+  hubAcceptsClient: true <2>
+----
+<1> Specify the name for the `ManagedCluster` resource.
+<2> Specify `true` to enable {rh-rhacm} to mange the cluster.
+
+.. Create the `ManagedCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-managed.yaml
+----
+
+.Verification
+
+* Check the status of the `ImageClusterInstall` and `nodes` resources in the hub cluster to monitor the progress of the target host installation by running the following command:
```

Comment:
`nodes` resources in the hub don't mean anything for this process.
You can check `nodes` in the spoke though once you can connect to the SNO.

Reply:
If you had example output for `watch oc get imageclusterinstall`, or even without the watch part, that would be great. But that's a general request for all commands in the deployment section as I don't have a configured environment.